File: 2765.py
n = input()
primeira = ''
segunda = ''
i = 0
while True:
    if n[i] != ',':
        primeira+=n[i]
    else:
        break
    i += 1
for l in range(n.index(',')+1, len(n)):
    segunda+=n[l]
print(primeira)
print(segunda)

File: 2765.js
const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const lines = input.split('\n');

let n = lines.shift();
let a = n.indexOf(',');
let primeira = ''; let segunda = '';
for(var i = 0; i < n.length; i++){
    if(i < a){
        primeira+=n[i];
    }else if(i > a){
        segunda+=n[i];
    }
}
console.log(primeira);
console.log(segunda);
